src/coreclr/jit/emitxarch.cpp

```diff
@@ -2569,6 +2321,152 @@ inline bool hasCodeMR(instruction ins)
     return ((insCodesMR[ins] != BAD_CODE));
 }
 
+//------------------------------------------------------------------------
+// emitGetVexPrefixSize: Gets Size of VEX prefix in bytes
+//
+// Arguments:
+//    id   -- The instruction descriptor
+//
+// Returns:
+//    Prefix size in bytes.
+//
+unsigned emitter::emitGetVexPrefixSize(instrDesc* id)
```

Comment:
This function had to be moved "down" so it could access `hasCodeMR`.

Like I mentioned above, I was originally going to go a different route but settled on the simpler approach here where we switch on the `insFmt` and do the couple minor checks rather than trying to build up the full `code_t`.

If we were caching the `code_t` somewhere so we didn't need to rebuild it 2-3 times, then another approach would be better. That is also a much more involved/complex change, but one that may be worthwhile long term.